server/lib/resolvers/wrapResolverWithHooks.js

```diff
@@ -0,0 +1,36 @@
+const _ = require('lodash')
+const {log} = require('../util/logger')
+const request = require('request')
+
+exports.wrapResolverWithHooks = function wrapResolverWithHooks (resolver, resolverMapping) {
+  return wrapResolverWithMappingHooks(resolver, resolverMapping)
+}
+
+function wrapResolverWithMappingHooks (resolverFn, resolverMapping) {
+  return (obj, args, context, info) => {
+    return new Promise(async (resolve, reject) => {
+      if (resolverMapping.preHook && !_.isEmpty(resolverMapping.preHook)) {
+        log.info(resolverMapping.preHook)
+        try {
+          request.get(resolverMapping.preHook)
```

Comment:
Do we want to send any payload to the webhook?

Reply:
@danielpassos request should use post method and for preHook it should send variables and for postHook should send response as it's body.